compiler/src/dotty/tools/dotc/ast/untpd.scala

```diff
@@ -281,9 +281,10 @@ object untpd extends Trees.Instance[Untyped] with UntypedTreeInfo {
       else {
         if (ms.nonEmpty)
           for (m <- ms)
-            assert(flags.isAllOf(m.flags) ||
-                   m.isInstanceOf[Mod.Private] && !privateWithin.isEmpty,
-                   s"unaccounted modifier: $m in $this when adding $ms")
+            assert(flags.isAllOf(m.flags)
+                || m.isInstanceOf[Mod.Private] && !privateWithin.isEmpty
+                || m.isInstanceOf[Mod.Abstract] || m.isInstanceOf[Mod.Override] && flags.is(AbsOverride),
```

Comment:
```suggestion
                || (m.isInstanceOf[Mod.Abstract] || m.isInstanceOf[Mod.Override]) && flags.is(AbsOverride),
```

Otherwise this assert will allow for an abstract modifier that is not reflected by an `AbsOverride` flag.

Reply:
Or maybe the actual fix should be in `flags.isAllOf`, to account for the `AbsOverride` being an umbrella over `Abstract` and `Override`?